pramen-py/tests/conftest.py

```diff
@@ -121,12 +135,15 @@ def load_and_patch_config(
     object.__setattr__(
         config.metastore_tables[0],
         "path",
-        create_parquet_data_stubs[0].resolve().as_posix(),
+        pathlib.Path(create_data_stubs_and_paths["parquet"])
```

Comment:
I think it is better to parametrize this fixture, so it is possible to say which format to use.
For example, something like:
```python
@pytest.mark.metastore(format="delta")
def test_foo(load_and_patch_config):...
```
That's easy to achieve, see for example this https://jaketrent.com/post/pass-params-pytest-fixture/